domain-ee/ee-ep-merge-app/src/python_src/util/contentions_util.py

```diff
@@ -7,9 +8,22 @@ def __init__(self, message=None):
         super().__init__(self.message)
 
 
+class CompareException(BaseException):
+    def __init__(self, message=None):
+        self.message = message
+        super().__init__(self.message)
+
+
 class ContentionsUtil:
     @staticmethod
     def merge_claims(pending_contentions: Response = None, ep400_contentions: Response = None):
         if pending_contentions and ep400_contentions:
-            return [*pending_contentions.contentions, *ep400_contentions.contentions]
+            return [*pending_contentions.contentions, *ContentionsUtil.new_contentions(pending_contentions.contentions, ep400_contentions.contentions)]
```

Comment:
If for some reason this method is used before any call to new_contentions, there's a possiblity of the `new_contentions` raising CompareException. Should it be caught here?